src/core/fft.jl

```diff
@@ -0,0 +1,106 @@
+import FFTW
```

Comment:
move determine_grid_size here? Also add a comment at the top saying that most of this is workarounds because FourierTransforms doesn't conform to AbstractFFTs (yet?)

Reply:
Agreed.